controller-server/src/test/java/com/yahoo/vespa/hosted/controller/ControllerTest.java

```diff
@@ -538,26 +539,6 @@ public void testGlobalRotations() throws IOException {
         assertTrue(rotationStatus.get("qrs-endpoint").getReason().equals("Testing I said"));
     }
 
-    @Test
-    public void testDeployUntestedChangeFails() {
```

Comment:
Shouldn't this still be tested?

Reply:
Triggering failure in this test is now impossible unless you manipulate things you shouldn't.